common/src/main/java/org/jboss/gm/common/alignment/Module.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ * <p>
+ * Licensed under the Eclipse Public License version 1.0, available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.jboss.gm.common.alignment;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.commonjava.maven.atlas.ident.ref.ProjectVersionRef;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * @author <a href="claprun@redhat.com">Christophe Laprun</a>
+ */
+public class Module {
```

Comment:
I take it this was broken out in order to allow `AlignedProject` to extend from it, correct?

Reply:
Yes.

Reply:
OK